app/renderer/components/navigation/publisherToggle.js

```diff
@@ -75,6 +75,7 @@ class PublisherToggle extends React.Component {
   }
 
   render () {
+    console.log(this.props)
```

Comment:
should this be removed?

Reply:
correct I already removed it, just didn't push it yet :)